spark/src/main/kotlin/com/adevinta/spark/components/icons/IconToggleButton.kt

```diff
@@ -203,7 +203,7 @@ public fun FilledTonalIconToggleButton(
     onCheckedChange: (Boolean) -> Unit,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
-    shape: Shape = SparkTheme.shapes.full,
```

Comment:
This api need to use the Shape instead of the ButtonShape otherwise it'll break usages in consumers